src/librustc_trans/back/lto.rs

```diff
@@ -22,6 +22,7 @@ use flate;
 
 use std::ffi::CString;
 use std::mem;
+#[allow(deprecated)]
```

Comment:
I removed all of them that could easily be replaced with `One`/`Zero` or were needed only for methods (afaik).